modal/functions.py

```diff
@@ -1229,16 +1276,16 @@ async def _call_generator(self, args, kwargs):
     async def _call_generator_nowait(self, args, kwargs):
         return await _Invocation.create(self.object_id, args, kwargs, self._client)
 
-    @warn_if_generator_is_not_consumed
-    @live_method_gen
-    @synchronizer.no_input_translation
-    async def map(
+    # note that `map()` is not synchronicity-wrapped, since it accepts executable code in the form of
```

Comment:
Should this have `synchronizer.nowrap` on it?

Reply:
Since it's a non-async function it's not strictly needed (it won't get executed in the synchronicity event loop regardless)